specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/examples/createMcppPurchase.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+      "resourceGroupName": "McppPurchases",
+      "api-version": "2023-02-09-preview",
+      "subscriptionId": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05",
+      "resourceName": "microsoftactionpack",
+      "body": { 
+        "productCode": "test product"            
+    }
+  },          
+  "responses": {
+    "200": {
```

Comment:
Are you sure the RP will be returning synchronously in most cases? I would've expected a 202 async response, and for the caller to keep polling until the operation is done.

Reply:
yes, its mostly async as we are waiting for createEvent , will change response to 202